packages/api/models/rules.js

```diff
@@ -227,6 +233,11 @@ class Rule extends Manager {
       }
       break;
     }
+    case 'sqs':
+      if (!(await aws.sqsQueueExists(newRuleItem.rule.value))) {
```

Comment:
`sqsQueueExists()` takes a queue name. When you give it the QueueUrl, it's going to say the queue does not exist because it's trying to look up the url by name. We might need another function for checking for existence here.

Reply:
`sqsQueueExists()` takes sqs url or name.

Reply:
Ok I figured out what the problem is. I did not have permissions setup for my queue. I think we need to change the error message (I'll go put a comment there) and add a note to the documentation (I'll put a comment there too).